src/main/java/org/broadinstitute/hellbender/utils/test/TargetsToolsTestUtils.java

```diff
@@ -77,7 +72,8 @@ public static SimpleInterval createOverEntireContig(final String contig) {
      * @return never {@code null}.
      * @throws UserException if there was some problem when creating the location.
      */
-    public static SimpleInterval createInterval(final String contig, final int start) {
+    public SimpleInterval createInterval(final String contig, final int start) {
+        // TODO: should this really be createInterval(contig, start, start) instead of using the constructor supplied here?
```

Comment:
I agree with you, this method is kindof pointless, it seems to only be called in one place and furthermore it seems to be used in one place, and given the the other overload it seems confusing to just use SimpleInterval

Reply:
I removed the TODO and call the `createInterval(contig, start, start)` instead, to be sure that it is using the sequence dictionary.